### PR TITLE
fix: Minor Issues

### PR DESCRIPTION
**Pull request by Jchinmay654**

### CODE REVIEW

File: .github/workflows/tf-vars.yml [hunk 1]
```diff
@@ -27,27 +24,33 @@ jobs:
...
-            -H "Authorization: Bearer ${{ inputs.GH_PAT }}" \
-            -H "Content-Type: application/json" \
-            -d '{"body": "Terraform Cleaner Output:\n\n'"${{ steps.terraform-cleaner.outputs['cleaner-output'] }}"'"}' \
-            "https://api.github.com/repos/${{ github.repository }}/issues/${{ steps.pr-number.outputs.pr-number }}/comments"
-
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          append: true
+          message: |
+            Test with ${{ github.sha }}.
```
Comment (maddy023) [inline .github/workflows/tf-vars.yml:53]: Update to "Output of Terraform Cleaner"

File: .github/workflows/tf-vars.yml [hunk 2]
```diff
@@ -27,27 +24,33 @@ jobs:
...
+            ```
```
Comment (maddy023) [inline .github/workflows/tf-vars.yml:54]: Remove this white space

File: .github/workflows/tf-vars.yml [hunk 3]
```diff
@@ -27,27 +24,33 @@ jobs:
...
+          output="${output//$'\r'/'%0D'}"
+          
+      - name: Check Cleaner Output
         run: |
-          echo "::set-output name=pr-number::$(jq --raw-output .pull_request.number $GITHUB_EVENT_PATH)"
-
+         echo "Cleaner Output:"
+         echo $output
+         
+      - name: Get PR number
```
Comment (maddy023) [inline .github/workflows/tf-vars.yml:44]: remove this step , no need anymore

File: .github/workflows/tf-vars.yml [hunk 4]
```diff
@@ -27,27 +24,33 @@ jobs:
...
         run: |
           mv terraform-cleaner /usr/local/bin
-          terraform-cleaner
-          cleaner_output=$(terraform-cleaner 2>&1)
-          echo "::set-output name=cleaner-output::$cleaner_output"
-
-      - name: Get PR Number
-        id: pr-number
+          echo "output=$(echo $(terraform-cleaner))" >> $GITHUB_ENV
+          output="${output//'%'/'%25'}"
```
Comment (maddy023) [inline .github/workflows/tf-vars.yml:35]: Update the output block if not remove these

File: .github/workflows/tf-vars.yml (hunk 2 above)
Comment (Jchinmay654) [reply .github/workflows/tf-vars.yml:54]: No space is there.

File: .github/workflows/tf-vars.yml (hunk 4 above)
Comment (Jchinmay654) [reply .github/workflows/tf-vars.yml:35]: output="${output//'%'/'%25'}"
          output="${output//$'\n'/'%0A'}"
          output="${output//$'\r'/'%0D'}"
          these output block you are talking about ...?

File: .github/workflows/changelog.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+name: 'CI'
```
Comment (maddy023) [inline .github/workflows/changelog.yml:1]: this should be workflow_on or on?

**Pull request merged**